bin/pycbc_live

```diff
@@ -280,106 +379,14 @@ class LiveEventManager(object):
-                    hdfp['f_end'] = f_end
-                    hdfp['sample_rate'] = bank.sample_rate
-                    hdfp['flow'] = bank.table[template_id].f_lower
-                    hdfp['mass1'] = bank.table[template_id].mass1
-                    hdfp['mass2'] = bank.table[template_id].mass2
-                    hdfp['spin1z'] = bank.table[template_id].spin1z
-                    hdfp['spin2z'] = bank.table[template_id].spin2z
-                    hdfp['template_duration'] = \
-                        bank.table[template_id].template_duration
-                    hdfp['ifar'] = ifar
-                    if gid is not None:
-                        hdfp['gid'] = gid
-
-                    for ifo in args.channel_name:
-                        hdfp['channel_names/{}'.format(ifo)] = \
-                            args.channel_name[ifo]
-
-                    recursively_save_dict_contents_to_group(hdfp,
-                                                            'mc_area_args/',
-                                                            self.mc_area_args)
-
-                cmd += '--params-file {} '.format(curr_fname)
-                cmd += '--approximant {} '.format(apr)
-                cmd += '--gracedb-server {} '.format(args.gracedb_server)
-                cmd += '--gracedb-search {} '.format(args.gracedb_search)
-                if not self.gracedb_testing:
-                    cmd += '--production '
-                cmd += '--verbose '
-                cmd += '--output-path {} '.format(self.path)
-                if self.enable_gracedb_upload:
-                    cmd += '--enable-gracedb-upload '
-
-                cmd += '--cores {} '.format(self.fu_cores)
-                if args.processing_scheme:
-                    # we will use the cores for multiple workers of the
-                    # optimization routine, so we force the processing scheme
-                    # to a single core here.  This may be enforcing some
-                    # assumptions about the optimal way to do FFTs on the
-                    # machine. However, the dominant cost of pycbc_optimize_snr
-                    # is expected to be in waveform generation, which is
-                    # unlikely to benefit from a processing scheme with more
-                    # than 1 thread anyway.
-                    opt_scheme = args.processing_scheme.split(':')[0]
-                    cmd += '--processing-scheme {}:1 '.format(opt_scheme)
-
-                log_fname = fname.replace('.xml.gz', '_optimize_snr.log')
-
-                logging.info('Running %s with log to %s', cmd, log_fname)
-                with open(log_fname, "w") as logfile:
-                    subprocess.Popen(
-                        cmd, shell=True, stdout=logfile, stderr=logfile
-                    )
+            if self.run_snr_optimization \
+                    and self.ifar_upload_threshold < ifar:
+                self.setup_optimize_snr(coinc_results, coinc_ifos, bank,
```

Comment:
should the change from `live_ifos` to `coinc_ifos` have any effect?

Reply:
I had tried to keep the original live/coinc ifos the same as before (as I wasn't sure on the differences), but it looks like I didn't succeed

Reply:
TBH it's not clear if this could be causing an error, but I think worth checking anyway. Maybe Ian can give you some hints on what the different ifo lists are all doing

Reply:
OK, if this is a problem is someone planning to open an issue or PR?

Reply:
Yes, I am at a conference this week, and busy for the next couple, but hope to get to this in the coming weeks

Reply:
I am looking into this.

Reply:
See #4086, closing this conversation.